node_helper.js

```diff
@@ -39,7 +39,7 @@ module.exports = NodeHelper.create({
 		 * A great writeup + the code tool he mentions in the article. These have been a great help
 		 * to me.
 		 */
-		const currDate = new Date();
+		const currDate = new Date(Date.now());
```

Comment:
I'm curious about this pattern - I don't think I've seen it used before.

What is the benefit to using this versus just `new Date()`? As far as I can tell, they both return the current datetime.